examples/simple-web-cluster/src/main/resources/visitors-creation-script.sql

```diff
@@ -19,18 +19,22 @@
 create database visitors;
 use visitors;
 
-# not necessary to create user if we grant (and not supported in some dialects)
-# create user 'brooklyn' identified by 'br00k11n';
-
-grant usage on *.* to 'brooklyn'@'%' identified by 'br00k11n';
-
-# ''@localhost is sometimes set up, overriding brooklyn@'%', so do a second explicit grant
-grant usage on *.* to 'brooklyn'@'localhost' identified by 'br00k11n';
 
+# the below will create user (and note create user not supported in some dialects)
+grant usage on *.* to 'brooklyn'@'%' identified by '${config["creation.script.password"]}';
```

Comment:
As I mentioned on the other PR, this is script is used quite a few different places [here](https://brooklyn.apache.org/v/latest/blueprints/multiple-services.html) for example. There should probably be a default so it doesn't break those places.